bsoid_app/extract_features.py

```diff
@@ -149,44 +150,43 @@ def learn_embeddings(self):
         np.random.seed(0)
         self.sampled_features = features_transposed[np.random.choice(features_transposed.shape[0],
                                                                      self.train_size, replace=False)]
-        st.info('Randomly sampled **{} minutes**... '.format(self.train_size / 600))
+        print('Randomly sampled {} minutes... '.format(self.train_size / 600))
         mem = virtual_memory()
         available_mb = mem.available >> 20
-        st.write('You have {} MB RAM 🐏 available'.format(available_mb))
+        print('You have {} MB RAM 🐏 available'.format(available_mb))
         if available_mb > (sampled_input_feats.shape[0] * sampled_input_feats.shape[1] * 32 * 60) / 1024 ** 2 + 64:
-            st.write('RAM 🐏 available is sufficient')
+            print('RAM 🐏 available is sufficient')
```

Comment:
I might suggest removing the emojis. while it might be fine, I imagine these might cause issues on certain run systems?

Reply:
🐏 has been ram-oved.